src/Sentry/PlatformAbstractions/RuntimeInfo.cs

```diff
@@ -8,7 +8,7 @@ namespace Sentry.PlatformAbstractions
     // https://github.com/dotnet/corefx/issues/17452
     internal static class RuntimeInfo
     {
-        private static readonly Regex RuntimeParseRegex = new("^(?<name>[^\\d]*)(?<version>(\\d+\\.)+[^\\s]+)",
+        private static readonly Regex RuntimeParseRegex = new("^(?<name>[a-zA-Z0-9 ]+)(?<version> [0-9\\.]+)? \\(.*$",
```

Comment:
```suggestion
        private static readonly Regex RuntimeParseRegex = new("^(?<name>[\w ]+)(?<version> [0-9\\.]+)? \\(.*$",
```

How about `\w`: https://docs.microsoft.com/en-us/dotnet/standard/base-types/character-classes-in-regular-expressions#word-character-w ?